packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/jni/JSIInteropModuleRegistry.kt

```diff
@@ -14,16 +16,21 @@ class JSIInteropModuleRegistry(appContext: AppContext) {
   @DoNotStrip
   private val mHybridData = initHybrid()
 
-  @Suppress("KotlinJniMissingFunction")
   private external fun initHybrid(): HybridData
 
-  @Suppress("KotlinJniMissingFunction")
   external fun installJSI(
     jsRuntimePointer: Long,
     jsInvokerHolder: CallInvokerHolderImpl,
     nativeInvokerHolder: CallInvokerHolderImpl
   )
 
+  external fun installJSIForTests()
+
+  @Throws(JavaScriptEvaluateException::class)
+  external fun evaluateScript(script: String): JavaScriptValue
+
+  external fun global(): JavaScriptObject
```

Comment:
I'd add the C++ comments in Kotlin too, as these functions will be mostly called from here
```suggestion
  /**
   * Evaluates given JavaScript source code.
   * @throws JavaScriptEvaluateException if the input format is unknown or evaluation causes an error
   */
  @Throws(JavaScriptEvaluateException::class)
  external fun evaluateScript(script: String): JavaScriptValue

  /**
   * Returns the runtime global object
   */
  external fun global(): JavaScriptObject
```